src/conductor/op-conductor/launcher.star

```diff
@@ -93,26 +107,23 @@ def get_service_config(
             params.service_name, consensus_port.number
         ),
         "OP_CONDUCTOR_CONSENSUS_PORT": str(consensus_port.number),
-        "OP_CONDUCTOR_EXECUTION_RPC": sidecar_context.rpc_http_url
-        if sidecar_context
-        else _net.service_url(
-            el_params.service_name, el_params.ports[_net.RPC_PORT_NAME]
+        "OP_CONDUCTOR_EXECUTION_RPC": (
+            sidecar_context.rpc_http_url
+            if sidecar_context
+            else _net.service_url(
+                el_params.service_name, el_params.ports[_net.RPC_PORT_NAME]
+            )
```

Comment:
Did linter produced this change? Or was it failing on this? Usually linter takes care of these things, wonder why it would change now